### PR TITLE
Feature/sett oppgave beskrivelse

### PR DESCRIPTION
**Pull request by MengZhuNAV**
Set the "beskrivelse" of oppgave based on the title of the main document of the journalpost.
See: https://favro.com/organization/98c34fb974ce445eac854de0/1844bbac3b6605eacc8f5543?card=Tea-1210

### CODE REVIEW

File: src/main/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveMapper.kt [hunk 1]
```diff
@@ -53,4 +53,9 @@ class OppgaveMapper(private val aktørClient: AktørClient) {
         if (journalpost.dokumenter.isNullOrEmpty()) throw error("Journalpost ${journalpost.journalpostId} mangler dokumenter")
         return if (journalpost.dokumenter.any { it.brevkode == "NAV 33-00.15" }) Behandlingstype.Utland.value else null
     }
+
+    private fun hentHovedDokumentTittel(journalpost: Journalpost): String? {
+        if (journalpost.dokumenter.isNullOrEmpty()) throw error("Journalpost ${journalpost.journalpostId} mangler dokumenter")
```
Comment (henninghaakonsen) [inline src/main/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveMapper.kt:58]: You don't need throw when using error.

File: src/test/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveClientTest.kt [hunk 2]
```diff
@@ -12,6 +12,7 @@ import no.nav.familie.kontrakter.felles.objectMapper
 import no.nav.familie.kontrakter.felles.oppgave.OppgaveResponse
 import no.nav.familie.kontrakter.felles.oppgave.Oppgavetype
 import no.nav.familie.log.NavHttpHeaders
+import no.nav.familie.prosessering.TaskStepBeskrivelse
```
Comment (henninghaakonsen) [inline src/test/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveClientTest.kt:15]: This is not in use.

File: src/main/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveMapper.kt (hunk 1 above)
Comment (MengZhuNAV) [reply src/main/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveMapper.kt]: Yes, it was copy-paste... Fixed

File: src/test/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveClientTest.kt (hunk 2 above)
Comment (MengZhuNAV) [reply src/test/kotlin/no/nav/familie/ba/mottak/integrasjoner/OppgaveClientTest.kt]: Ok

**Pull request merged**